docs/PublishingReleases.md

```diff
@@ -1,20 +0,0 @@
-[lerna]: https://github.com/lerna/lerna
-[from-package]: https://github.com/lerna/lerna/tree/master/commands/publish#bump-from-package
-
-Publishing Releases
-===================
-
-We're using [lerna][lerna] to make managing multiple packages in the same repos easier.
-
-Before releasing a new version, it is a good idea to release a prerelease so that folks can test out the changes.
-
-## Publishing a new version
-
-1. Run `yarn publish-release`. The publish command will handle bumping the version, tagging via git, and publishing the packages to NPM.
-1. If there are publishing errors, you can run again via `yarn lerna publish from-package` ([Docs][from-package]).
-
-## Publishing a prerelease
-
-1. Run `yarn publish-prerelease`. The publish command will handle bumping the version, tagging via git, and publishing the packages to NPM under the `next` dist-tag.
-1. If there are publishing errors, you can run again via `yarn lerna publish from-package --dist-tag next` ([Docs][from-package]).
-
```

Comment:
I was going to update this with the new release commands, but wasn't sure if it even made a ton of sense to have release documentation on a public repo

Reply:
Yeah I'm fine with removing this 👍 we can keep it internal